backend_pt.md

```diff
@@ -0,0 +1,75 @@
+# Desafio de Backend
+> *This file is also available in english [here](backend.md).*
+
+No Zé, encontramos o melhor parceiro para entregar as bebidas aos nossos consumidores, oferecendo o melhor e mais rápido serviço de entrega de bebidas.
+Para isso nossos computadores lidam o tempo todo com objetos [GIS](https://en.wikipedia.org/wiki/Geographic_information_system).
```

Comment:
```suggestion
Para isso nossas máquinas lidam o tempo todo com objetos [GIS](https://en.wikipedia.org/wiki/Geographic_information_system).
```

Reply:
Why do you think `máquinas` is a better translation to `compute fleet`?

Reply:
how about `Servidores`?

Reply:
`servidores` would be nice but would also be a lie... We run `server-less` 🤣.

Reply:
sei la, computador me remete a alguma coisa antiga, mas eh soh um sentimento kkkk